scripts/wrist_test.py

```diff
@@ -0,0 +1,245 @@
+               'head_axis0', 'head_axis1', 'head_axis2',
+               'shoulder_right_axis0', 'shoulder_right_axis1', 'shoulder_right_axis2',
+               'elbow_right_axis0', 'elbow_right_axis1',
+               'wrist_right_axis0', 'wrist_right_axis1', 'wrist_right_axis2']
+
+
+def hard_reset():
+    msg = JointState()
+
+    msg.name = joints_name
+    msg.position = [0] * len(msg.name)
+    msg.velocity = [0] * len(msg.name)
+    msg.effort = [0] * len(msg.name)
+    publisher.publish(msg)
+
+    rospy.loginfo("Hard reset")
+    rospy.sleep(2.0)
+
+
+def hand_shake(left_or_right="right", ranges=None):
+    if ranges is None:
+        ranges = [0.6, 1.0]
+
+    msg = JointState()
+    msg.name = ["elbow_" + left_or_right + "_axis0"]
+    msg.velocity = [0] * len(msg.name)
+    msg.effort = [0] * len(msg.name)
+
+    samples = 50
+    for i in range(10):
+        if not i % 2:
+            msg.name = ["elbow_" + left_or_right + "_axis0"]
+            el_axis1_targets = np.linspace(ranges[0], ranges[-1], samples)
+
+            for t2 in el_axis1_targets:
+                msg.name = ["elbow_" + left_or_right + "_axis0"]
+                msg.position = [t2]
+                msg.velocity = [0]
+                msg.effort = [0]
+                publisher.publish(msg)
+                rate.sleep()
+        else:
+
+            el_axis1_targets = np.linspace(ranges[-1], ranges[0], samples)
+
+            for t2 in el_axis1_targets:
+                msg.name = ["elbow_" + left_or_right + "_axis0"]
+                msg.position = [t2]
+                msg.velocity = [0]
+                msg.effort = [0]
+                publisher.publish(msg)
+                rate.sleep()
+
+
+def move_shoulder_elbow_wrist(left_or_right="right", move_elbow=True, move_wrist=True,
+                              shoulder_range=None, elbow_range=None, wrist_range=None):
+    shoulder_range = [0.0, -1.1] if not shoulder_range else shoulder_range
+    elbow_range = [0.0, 0.6] if not elbow_range else elbow_range
+    wrist_range = [-1.5, -0.05] if not wrist_range else wrist_range
+
+    msg = JointState()
+    num_samples = 200
+
+    msg.name = ["shoulder_" + left_or_right + "_axis0"]
+    targets = [np.linspace(shoulder_range[0], shoulder_range[1], num_samples)]
+    if move_elbow:
+        msg.name.append("elbow_" + left_or_right + "_axis0")
+        targets.append(np.linspace(elbow_range[0], elbow_range[1], num_samples))
+    if move_wrist:
+        msg.name.append("wrist_" + left_or_right + "_axis0")
+        targets.append(np.linspace(wrist_range[0], wrist_range[1], num_samples))
+
+    targets = np.array(targets)
+
+    for i in range(num_samples):
+        msg.position = list(targets[:, i])
+        msg.velocity = [0] * len(msg.name)
+        msg.effort = [0] * len(msg.name)
+        publisher.publish(msg)
+        rate.sleep()
+
+
+def move_wrist(left_or_right="right", axis=0, ranges=None):
+    ranges = [-1.5, -0.05] if not ranges else ranges
```

Comment:
```suggestion
def move_wrist(left_or_right="right", axis=0, ranges=[-1.5, -0.05]):

```

Reply:
hm, it doesnt make sense to put these ranges by default. either pick a range that would work for every joint in roboy or leave it empty
also see my edit how to assign values to params

Reply:
Done